.github/workflows/_github-release.yml

```diff
@@ -27,16 +27,19 @@ on:
   workflow_call:
     inputs:
```

Comment:
Please update the documentation at the top to match these inputs.

Reply:
Thanks for the point! Fixed in https://github.com/dbt-labs/dbt-release/commit/20a648cd3012d942169b6fca1b0b96159427eb3f.